src/data-sources/connections/application-insights/QueryTester.tsx

```diff
@@ -102,15 +111,9 @@ export default class QueryTester extends React.Component<IQueryTesterProps, IQue
             onChange={this.onQueryChange}
           />
           <Divider />
-          <TextField
-            id="responseView"
-            block
-            paddedBlock
-            rows={4}
-            defaultValue={response}
-            disabled
-            inputStyle={{ 'max-height': responseHeight }}
-          />
+          <div style={{overflowY: 'scroll', height: '75%', width: '100%', position: 'absolute'}}>
+            <JSONTree data={response} theme="default" shouldExpandNode={() => expandResponse}/>
```

Comment:
shouldExpandNode is defined as function in react-json-tree library.
It is a function because it supports the option to choose which node to expand (if passing the node name)